opm/input/eclipse/Schedule/Well/WellTestState.hpp

```diff
@@ -271,7 +276,7 @@ class WellTestState {
 private:
     std::unordered_map<std::string, WTestWell> wells;
     std::unordered_map<std::string, std::unordered_map<int, ClosedCompletion>> completions;
-
+    std::vector<std::string> follow_on_wells;
```

Comment:
I am not sure that I understand everything correct, so excuse me if I see things wrong. 

I think with item 9 of WECON, we establish a relationship/mapping between one well A and another well B (which should be SHUT). 

In case the well A is shut due to economic limits or physical limit, we open the well B. 

But if we put them in a std::vector for all the wells, how do we manage the relationship between the well A -> well B?

Reply:
Now I see that it is only added when the specific wells are shut. You can discard the above comment.